### ISSUE TITLE
[ISSUE] Console app takes forever to load in alpha >12 seconds

### ISSUE THREAD

**Issue by jessesimpson36** [camunda/camunda-platform-helm#2333]
**Describe the issue:**

When navigating to console in SM on alpha charts, it takes > 12 seconds to load

[demo.ci.distro.ultrawombat.com.har.txt](https://github.com/user-attachments/files/16966504/demo.ci.distro.ultrawombat.com.har.txt)

**Actual behavior:**

**Expected behavior:**

**How to reproduce:**

**Logs:**

**Environment:**

**Please note: Without the following info, it's hard to resolve the issue and probably it will be closed.**

- Platform: 
- Helm CLI version: 
- Chart version: 8.6 alpha snapshot
- Values file:

**Comment by jessesimpson36** [comment]
https://github.com/camunda/distribution/issues/306

**Comment by aabouzaid** [comment]
I've checked with the dev team, and the team stated that there have been no changes in the load time. I also reviewed it, which shows that the wait comes from the Console waiting for Identity/Keycloak to auth.

I've also tested it with a deployed chart, and it worked as expected (I deleted the Console Pod and measured the frontend boot time).

I've suggested to the Console team to provide some details in the UI so it enhances the UX, but in general, that's not a blocker or a bug.
